plugins/inputs/kafka_consumer/README.md

```diff
@@ -22,6 +22,9 @@ and use the old zookeeper connection method.
   ## Offset (must be either "oldest" or "newest")
   offset = "oldest"
 
+  ## Optional client id
+  # client_id = "my_client"
```

Comment:
Let's have the example ID be `Telegraf`?  Perhaps this should be the default instead of "sarama"?

Reply:
The default value of sarama is set by the underlying implementation used by both kafka plugins.  I can add the logic to set the id to "Telegraf" when a value is not supplied.